### PR TITLE
Enable SSL for all server

### PR DESCRIPTION
**Pull request by TasdidurRahman**
enable ssl connection with upstream. no ca used. 
variable introduced: 
`PMA_SSL` for single upstream
`PMA_SSLS` for multiple upstream
`docker run ... -e PMA_HOSTS="mysql-notls,mysql-tls" -e PMA_SSLS="0,1" ...`

### CODE REVIEW

**Comment by williamdes** [comment]
> wrong upstream

If you want you can contribute adding support for SSL via ENVs
See: https://github.com/phpmyadmin/docker/issues?q=is%3Aissue+is%3Aopen+SSL

**Comment by TasdidurRahman** [comment]
hi @williamdes, can you review this?

**Review by williamdes — CHANGES_REQUESTED** [review]
Please update https://github.com/phpmyadmin/docker/blob/master/config.inc.php
And run ./update.sh

Can you also add documentation on the README ?
It is unclear how to use this

`PMA_SSLS="0,1"` is maybe not very user friendly

**Comment by TasdidurRahman** [comment]
> PMA_SSLS="0,1" is maybe not very user friendly
imo this is better to keep it simple as we are not using CA rn. still we are open for suggestion! 
@williamdes

**Review by williamdes — CHANGES_REQUESTED** [review]
Okay, I did not see that other ENVs have the same logic
Please add documentation to the README and say the variable works in the same way that PMA_PORTS works

File: README.md [hunk 1]
```diff
@@ -191,6 +203,10 @@ In order to keep your sessions active between container updates you will need to
 
 For usage with Docker secrets, appending ``_FILE`` to the ``PMA_PASSWORD`` environment variable is allowed (it overrides ``PMA_PASSWORD`` if it is set):
 
+* ``PMA_SSL`` - define ssl usage for MySQL server
+
+* ``PMA_SSLS`` - comma separated list of 0 and 1 defining ssl usage for corresponding MySQL servers
```
Comment (williamdes) [inline README.md:208]:
```suggestion
* ``PMA_SSL`` - Enable ssl usage for the MySQL connection
* ``PMA_SSLS`` - comma separated list of `0` and `1` defining ssl usage for the corresponding MySQL connections
```
Please move this at the end of the list

Comment (TasdidurRahman) [reply README.md]: moved

File: README.md [hunk 2]
```diff
@@ -158,6 +158,18 @@ In order to keep your sessions active between container updates you will need to
 -v /some/local/directory/sessions:/sessions:rw
 ```
 
+## Connect over SSL with backend
+
+Set the variable ``PMA_SSL`` to '1' and enable ssl usage from phpmyadmin to mysql server. The default value is 0. Variable ``PMA_SSLS`` can be used as a comma seperated sequence of 0 and 1 where multiple hosts are mentioned. Values order must follow the ``PMA_HOSTS`` and will be computed accordingly.
```
[the remark this answers is not in the document]
Comment (TasdidurRahman) [reply README.md]: updated

**Review by MauricioFauth — APPROVED** [review]

**Review by williamdes — COMMENTED** [review]
Thank you !

**Comment by williamdes** [comment]
I sent your changes to the official registry: https://github.com/docker-library/official-images/pull/17398

File: README.md [hunk 3]
```diff
@@ -206,6 +220,8 @@ docker run --name phpmyadmin -d -e PMA_PASSWORD_FILE=/run/secrets/db_password.tx
 - `PMA_CONTROLHOST`
 - `PMA_CONTROLUSER`
 - `PMA_CONTROLPASS`
+- `PMA_SSL`
+- `PMA_SSLS`
```
Comment (williamdes) [inline README.md:224]: @TasdidurRahman I am considering a revert of the two lines
It makes no sense to read 1 and 0 from a file
What do you think ?

Comment (williamdes) [reply README.md:224]: Reverted as 577a94fda60fbd1bb13247531c1783db9c51a64b

Comment (TasdidurRahman) [reply README.md:224]: sorry for late reply, LGTM

Comment (williamdes) [reply README.md:224]: Thank you

**Pull request merged**